dashboard/views.py

```diff
@@ -6,7 +7,19 @@
 from push.models import PushApplication
 
 
-class Home(TemplateView):
+class Login(TemplateView):
+    template_name = 'dashboard/login.html'
+
+
+class Status(TemplateView):
+    template_name = 'dashboard/status.html'
+
+
+class PermissionDenied(TemplateView):
+    template_name = 'dashboard/errors/403.html'
+
+
+class Home(LoginRequiredMixin, TemplateView):
```

Comment:
We don't need the `LoginRequiredMixin` here do we? With it, as soon as I go to http://127.0.0.1:8000 I'm redirected to http://127.0.0.1:8000/accounts/login/?next=/